com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/jobs/FlowableOperationsCleaner.java

```diff
@@ -0,0 +1,51 @@
+package com.sap.cloud.lm.sl.cf.process.jobs;
+
+import java.text.MessageFormat;
+import java.util.Date;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.flowable.engine.runtime.ProcessInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.annotation.Order;
+
+import com.sap.cloud.lm.sl.cf.process.Messages;
+import com.sap.cloud.lm.sl.cf.process.flowable.FlowableFacade;
+import com.sap.cloud.lm.sl.cf.web.api.model.Operation;
+
+@Named
+@Order(30)
+public class FlowableOperationsCleaner implements Cleaner {
```

Comment:
Flowable uses "process" as a term and not "operation". Operations exist only in our codebase and we're not using the `OperationService` here, so the current name is a little confusing and misleading. I suggest renaming this class to `FlowableDataCleaner`, which will be consistent with `FlowableHistoricDataCleaner`.